src/scripts/servers.js

```diff
@@ -288,6 +291,23 @@ class Servers extends EventEmitter {
         });
     }
 
+    resetAppData () {
+        return remote.dialog.showMessageBox({
+            type: 'question',
+            buttons: ['Yes', 'Cancel'],
+            defaultId: 1,
+            title: 'Reset App Data',
+            message: 'This will sign you out from all your teams and reset the app back to its original settings. This cannot be undone.'
+        }, (response) => {
+            if (response === 0) {
+                const dataDir = remote.app.getPath('userData');
```

Comment:
The localStorage is built from the user data directory as well? Or should we clear that too?